types/types.go

```diff
@@ -706,6 +714,11 @@ func ErrorAlreadyExists(err error) bool {
 	return strings.Contains(err.Error(), "already exists")
 }
 
+func ErrorIsNotNow(err error) bool {
+	var dummy *ErrorNotNow
+	return errors.As(err, &dummy)
+}
+
```

Comment:
This casting logic is cool. I am wondering if you have any documents to learn more about this?

Reply:
This came from the docs for github.com/pkg/errors (not to be confused with the golang standard library errors package, which has superseded it, but is still compatible.)
Yeah, it's pretty cool, and better than the first thing I tried, which was a straight cast: err.(*ErrorNotNow).  That doesn't work for wrapped errors, but errors.As() will unwrap and check at each level.

Reply:
Cool. Thank you